### DISCUSSION TITLE
Templates are confusing

### DISCUSSION THREAD

**Discussion by Gavin-Williams** [microsoft/WindowsAppSDK#1771] [General] [answered]
There are two templates that look exactly the same to me. What's the difference between these two? 

> **Blank App, Packaged (WinUI 3 in Desktop)**
   A project template for creating a desktop app based on the Windows UI library (WinUI 3) along with a MSIX package for side-loading or distribution via Microsoft Store.

> **Blank App, Packaged with WAP (WinUI 3 in Desktop).**
    A project template for creating a desktop app based on the Windows UI library (WinUI 3). A Windows application packaging (WAP) project is included to create a MSIX package for side-loading or distribution via the Microsoft Store.
    
 And also, where is the Windows App SDK template for developing without Xaml? 
 
And under the third dropdown in the Create a new project tool 'All project types' - you have WinUI, but you don't have Windows App SDK while you do have Windows App SDK under the second dropdown 'All Platforms'. I think you have that wrong. Windows App SDK is not a platform, it's a framework on the Windows Platform. I think it needs to be shuffled over to the 3rd dropdown. 

Edit: And I wouldn't mind the ability to hide particular project templates so that I can simplify the view.

**Comment by riverar** [comment]
Agree, and I would promote this to an issue. I usually have to read the descriptions 100 times to pick the right one. (One creates a separate Packaging project and one does not, opting instead for the single project solution.)

**Comment by andrewleader** [comment, marked as answer]
This is great feedback, thanks @Gavin-Williams, we've captured your feedback (and Rafael's) and are tracking this with the following feature idea: https://portal.productboard.com/winappsdk/1-windows-app-sdk/c/52-windows-template-studio

Also, as for the incorrect platform, that's a known bug that we're fixing in 1.0.1: #1665
